### PR TITLE
Fixes #8474: Debian squeeze box should modify it's repo brefore making apt-get update

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/8474

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
PR updated

File: scripts/cleanbox.sh [hunk 1]
```diff
@@ -101,13 +108,6 @@ then
   fi
 
   # Replace repos by archive for Debian Squeeze
```
Comment (peckpeck) [inline scripts/cleanbox.sh]: You didn't move the accompanying comment

File: scripts/cleanbox.sh [hunk 2]
```diff
@@ -101,13 +108,6 @@ then
   fi
```
Comment (peckpeck) [inline scripts/cleanbox.sh]: It occurs to me that there should be an apt-get update just before this if too

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
PR updated

File: scripts/cleanbox.sh [hunk 3]
```diff
@@ -100,13 +108,7 @@ then
...
 
-  # Replace repos by archive for Debian Squeeze
-  grep -e "^6\." /etc/debian_version > /dev/null
-  squeeze=$?
-  if [ $squeeze -eq 0 ] ;
-  then
-    echo "deb http://archive.debian.org/debian/ squeeze main" > /etc/apt/sources.list
-  fi
+  apt-get update
```
Comment (peckpeck) [inline scripts/cleanbox.sh]: The update is not needed here, just within the if
I should have said just before the "fi"

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**